microovn/ovn/cmd/commands.go

```diff
@@ -132,7 +132,14 @@ func ovnDBCtl(ctx context.Context, s state.State, dbType OvsdbType, timeout int,
 		return "", err
 	}
 
-	return shared.RunCommandContext(ctx, baseCmd, args...)
+	// try command 3 times if it is failing
+	for attempts := 0; attempts < 3; attempts++ {
+		output, err := shared.RunCommandContext(ctx, baseCmd, append(args, "--timeout", "30")...)
+		if err == nil {
+			return output, nil
+		}
+	}
+	return "", err
```

Comment:
+1 for timeout here. I'm just wondering if it's necessary for us to retry it 3 times. OVS/OVN commands are pretty good at figuring out when retry helps and when not. For example if connection fails, `ovn-nbctl` is going to retry with increasing backoff. However if there's something like wrong argument, it will hard fail without retries. If our intention is to have 90s timeout, we can just put it in there directly, and let ovn figure out when retries are helpful and when not.

Note: there's a `VSCtl` function that executes `ovs-vsctl` and that might benefit from the `--timeout` argument as well.